config/runtime.exs

```diff
@@ -20,6 +20,14 @@ if System.get_env("PHX_SERVER") do
   config :videoroom, VideoroomWeb.Endpoint, server: true
 end
 
+if Mix.env() == :test do
+  # FIXME it seems that divo tries to do docker cleanup
+  # before RoomService exits, which results in a bunch of error
+  # logs at the end of tests - RoomService is linked to the
+  # Notifier and Notifier to the WS connection to the JF
+  Divo.Suite.start(services: [:jellyfish], auto_start: false)
+end
```

Comment:
That's definitely not perfect but it at least somehow works.

The problem is that it seems that divo tries to cleanup docker images before RoomService exits so we turn Jellyfish off, WS connection goes down and our RoomService crashes because it's linked to the Notifier, which results in a bunch of error logs. If we want to make it better, let's do this in another PR.

I can't put this in application.ex as Divo.Suite.start is private and dialyzer warns this fact

I can also entirely remove divo and make an rename mix integration_test to mix test so that we always test using docker compose.

Reply:
Personally, I rarely if ever use the Divo - I usually comment out the divo start and either use local Jellyfish or integration test. So I think we can remove Divo altogether and you can do it in this PR as well, since there aren't a lot of changes. WDYT @Rados13 ?

Reply:
I have no problem with removing Divo. To explain why I started using Divo, the idea was to have the possibility to run tests faster without building a docker image, which based on my observation, was a little bit slower, but this doesn't work well here, so removing it sounds like a good idea.